loadrunner_professional/assets/logs/loadrunner_professional.yaml

```diff
@@ -0,0 +1,71 @@
+id: loadrunner_professional
+metric_id: loadrunner_professional
+facets: null
+pipeline:
+  type: pipeline
+  name: LoadRunner Scenario Information Pipeline
+  enabled: true
+  filter:
+    query: "source:loadrunner "
+  processors:
+    - type: attribute-remapper
+      name: LoadRunner Hostname Tag Creation
+      enabled: true
+      sources:
+        - hostname
+      sourceType: attribute
+      target: lr_hostname
+      targetType: tag
+      preserveSource: false
+      overrideOnConflict: false
+    - type: attribute-remapper
+      name: LoadRunner Service Tag Creation
+      enabled: true
+      sources:
+        - service
+      sourceType: attribute
+      target: lr_service
```

Comment:
Why do we want to drop the `service` attribute and remap it to a custom `lr_service` tag ?

Reply:
removed the remapper entirely